src/tests/ffi-key-sig.cpp

```diff
@@ -109,6 +109,32 @@ TEST_F(rnp_tests, test_ffi_key_signatures)
     assert_rnp_success(rnp_key_get_signature_count(subkey, &sigs));
     assert_int_equal(sigs, 1);
     assert_rnp_success(rnp_key_get_signature_at(subkey, 0, &sig));
+
+    rnp_output_t output = NULL;
+    assert_rnp_success(rnp_output_to_memory(&output, 154));
+    assert_rnp_success(rnp_signature_export(sig, output, 0));
+    uint8_t *buf = NULL;
+    size_t   len = 0;
+    assert_rnp_success(rnp_output_memory_get_buf(output, &buf, &len, false));
+    assert_int_equal(len, 154);
+    assert_rnp_success(rnp_output_destroy(output));
+
+    output = NULL;
+    assert_rnp_success(rnp_output_to_memory(&output, 4096));
+    assert_rnp_success(rnp_signature_export(sig, output, RNP_KEY_EXPORT_ARMORED));
+    buf = NULL;
+    len = 0;
+    assert_rnp_success(rnp_output_memory_get_buf(output, &buf, &len, false));
+    assert_int_equal(len, 297);
+    std::string data((const char *) buf, len);
+    assert_true(starts_with(data, "-----BEGIN PGP PUBLIC KEY BLOCK-----"));
+    assert_true(ends_with(strip_eol(data), "-----END PGP PUBLIC KEY BLOCK-----"));
+
+    rnp_input_t input;
+    assert_rnp_success(rnp_input_from_memory(&input, buf, len, false));
+    assert_rnp_success(rnp_import_signatures(ffi, input, 0, NULL));
```

Comment:
I'm not sure if `rnp_import_signatures()` validates them during import, at least description does not explicitly say that. Maybe it is better to verify explicitly? @ni4 can you comment?

Also it would be nice to check the JSON output and make sure it looks as expected instead of passing NULL here.

Reply:
@antonsviridenko `rnp_import_signatures()` actually validates newly added sigs (see `rnp_key_store_import_key_signature()` -> `tmpkey.refresh_data()`)

+1 for JSON check.

Reply:
Argh, forget to address this. Will update the PR...

Reply:
Now it should be good to go.